### ISSUE TITLE
Installation changes existing pytorch build

### ISSUE THREAD

**Issue by eneskelestemur** [openmm/openmmforcefields#350]
I'm trying to install openmmforcefields and PyTorch in the same environment, but installing openmmforcefields through mamba tries to change the existing PyTorch-CUDA build to PyTorch-CPU build. Is there a way to avoid this change? I had the same issue with openff-toolkit, but they have the option to install only the base package with `openff-toolkit-base`, which does not try to to change existing PyTorch build. Is there something similar to this for openmmforcefields?

![image](https://github.com/user-attachments/assets/5c215795-79c2-41f2-a686-2cb0f5f644eb)

**Comment by mattwthompson** [comment]
Does this also happen when you install a fresh environment?

It's probably an AmberTools build issue, either way

**Comment by eneskelestemur** [comment]
Thank you for the quick response. I managed to make it work. I'm still not sure what the problem was, but after you mentioned `ambertools`, I installed it separately from `openmmforcefields` and `openff-toolkit`, and it worked. So, I installed the packages in the following order in a new environment: `openmm` with `cudatoolkit`, PyTorch (as described in the documentation), `ambertools`, `openmmforcefields`, `openff-toolkit`. 

Thanks again!
